README.md

```diff
@@ -144,15 +145,20 @@ Example:
 **Prerequisites**
 
 * JDK 8 or later 
-* Eclipse IDE
-
-**Importing the project into Eclipse**
-
-1. Open Eclipse
-2. Click `File` > `Import`
-3. Click `General` > `Existing Projects into Workspace` > `Next`
-4. Click `Browse`, then locate the project's directory
-5. Click `Finish`
+* IntelliJ IDE
+
+**Importing the project into IntelliJ**
+
+1. Open IntelliJ (if you are not in the welcome screen, click `File` > `Close Project` to close the existing project dialog first)
+3. Set up the correct JDK version
```

Comment:
Actually, we can put all `1.` and markdown will automatically number properly, but I don't think this PR should be concerned about that.